### PR TITLE
Arapaho paradigm support

### PR DESCRIPTION
**Pull request by andrewdotn**
This PR adds:
  - Labels and paradigms for Arapaho courtesy of @sarahrmoeller
  - Make the relabelling language-specific
  - Update the label-file-reformatter to reformat labels for all languages
  - Updated FSTs for Arapaho, including a spell-relaxed version
  - Initial `fstLemma` support, defaulting to off. There is a migration to add the column and populate the field. The migration will be run, but its outputs will not yet be used, by the Plains Cree dictionary.

### CODE REVIEW

**Comment by andrewdotn** [comment]
This PR does not add the following things, which are still to come:
  - Labels in the linguistic drop-down for Arapaho
  - Language-specific template files
  - Code to build a non-test-sized dictionary for Arapaho

**Review by eddieantonio — APPROVED** [review]
LGTM! I'm a bit confused about the migrations, but I'm assuming that's a development artifact.

File: src/morphodict/lexicon/migrations/0002_add_fst_lemma.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                null=True,
+                unique=True,
+            ),
+        ),
+        migrations.AddIndex(
+            model_name="wordform",
+            index=models.Index(
+                fields=["text", "raw_analysis"], name="lexicon_wor_text_cc8d78_idx"
+            ),
+        ),
```
[the remark this answers is not in the document]
Comment (andrewdotn) [reply src/morphodict/lexicon/migrations/0002_add_fst_lemma.py:72]: It turns out that *changing the `help_text`* [generates a migration](https://stackoverflow.com/questions/26503826/why-does-django-make-migrations-for-help-text-and-verbose-name-changes).

[the remark this answers is not in the document]
Comment (eddieantonio) [reply src/morphodict/lexicon/migrations/0002_add_fst_lemma.py:72]: TIL!

**Comment by andrewdotn** [comment]
Good to know, adding a column and traversing it in Python only took one minute:

    Jul 23 13:40:21 altlab-gw gunicorn[1268]: Operations to perform:
    Jul 23 13:40:21 altlab-gw gunicorn[1268]:   Apply all migrations: API, CreeDictionary, admin, auth, contenttypes, lexicon, sessions, sites
    Jul 23 13:40:21 altlab-gw gunicorn[1268]: Running migrations:
    Jul 23 13:41:00 altlab-gw gunicorn[1268]:   Applying lexicon.0002_add_fst_lemma... OK
    Jul 23 13:41:28 altlab-gw gunicorn[1268]:   Applying lexicon.0003_populate_fst_lemma... OK
    Jul 23 13:41:28 altlab-gw gunicorn[1268]: itwewina: successfully redeployed

**Pull request merged**